### DISCUSSION TITLE
Mac `Alt` (Opt) keybindings

### DISCUSSION THREAD

**Discussion by blogabe** [zellij-org/zellij#3095] [General]
I'm a Mac iTerm2 user and came across the keybinding fix at https://www.clairecodes.com/blog/2018-10-15-making-the-alt-key-work-in-iterm2/

Following these steps for every `Alt` key combo seems tedious and error prone. Has anyone come across a cleaner alternative?
Thanks.

**Comment by FeliciousX** [comment]
why does this happen in the first place? im also encountering this issue on Alacritty when using zellij.

**Comment by merikan** [comment]
https://zellij.dev/documentation/faq#i-am-a-macos-user-how-can-i-use-the-alt-key
